yang/frr-isisd.yang

```diff
@@ -1495,6 +1525,15 @@ module frr-isisd {
               "IS-IS level into which the routes should be redistributed.";
           }
 
+	  leaf level_from {
```

Comment:
The common practice in YANG models is to use dashes (`-`), not underlines (`_`). Please, change this to `level-from`.